test/libsolidity/semanticTests/storage/storage_boundary_array_copy.sol

```diff
@@ -0,0 +1,41 @@
+contract C {
+	constructor() {
```

Comment:
Some of your tests are indented with tabs, some with spaces. Let's not make things unnecessarily complicated and stick to one (spaces).
```suggestion
    constructor() {
```